sagemaker_neo_compilation_jobs/gluoncv_yolo_darknet/gluoncv_yolo_darknet_neo.ipynb

```diff
@@ -13,9 +13,10 @@
     "  2. [Convert data into RecordIO](#Convert-data-into-RecordIO)\n",
```

Comment:
<p>Please change "<strong><em>This is notebook is for demostration purpose only." </em></strong>to<strong><em> "This notebook is for demonstration purpose only."</em></strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Good catch, thanks :)